combat-trainer.lic

```diff
@@ -384,6 +384,12 @@ class LootProcess
     @dissect_priority = skinning['dissect_priority']
     echo("  @dissect_priority: #{@dissect_priority}") if $debug_mode_ct
 
+    # Setting to determine whether to always arrange, regardless of skinning, or dissecting
+    # Depending on the creature level, and the character's skinning skill, simply arranging will teach skinning
+    # Defaulting to true to keep existing behaviour.
+    @always_arrange = skinning['always_arrange'].nil? ? true : skinning['always_arrange']
```

Comment:
I saw this used to be `@always_arrange = skinning['always_arrange'] || true`, which I think is easier to read (and we use the same style below). Why was it changed?

Reply:
I want to say "because Seped told me to" but we found that this statement was always truthy.